coronavirus.md

```diff
@@ -0,0 +1,85 @@
+## Coronavirus (COVID-19)
+
+Information for Farset Labs members and visitors during the Coronavirus pandemic, in line with [government guidelines](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/our-approach-to-decision-making-summary-120520.pdf).
+
+### Status
+
+**The space is closed aside from maintenance visits by the management team, and all events are remote.**
+
+* **❌ Visitors: No**
+* **❌ Members: No**
+* **💻 Events and meetings: Remote**
+
+_Last updated on the 19th of May 2020._
+
+---
+
+### Asked Questions
```

Comment:
FAQs

Reply:
Can you add some reasoning, please?

Reply:
Not that I'm personally prescriptive in anyway but it's an already well established phrase/initialism.

Reply:
Ah, makes sense. I'll update the heading.